### ISSUE TITLE
Gutter click or double-click emit dragStart event, but not emit dragEnd event.

### ISSUE THREAD

**Issue by Mike-Jia** [bertrandg/angular-split#186]
**Describe the bug**
Gutter click or double-click emit dragStart event, but not emit dragEnd event. So, in examples iframe, just click the gutter, hider div would show and block iframes until drage the gutter(at least move 1 px).

version: 3.0.2

**To Reproduce**
Create a Stackblitz demoing the bug/issue:
https://bertrandg.github.io/angular-split/#/examples/iframes

**Comment by Mike-Jia** [comment]
Hi @bertrandg ,
  Do you need more information? Any plan to fix this?

**Comment by karedas** [comment]
same here

**Comment by eranshmil** [comment]
Here is the workaround I'm using:
```
  (gutterClick)="splitGutterClick($event)"
```

```ts
  import { IOutputData } from 'angular-split/lib/interface';

  public splitGutterClick({ gutterNum }: IOutputData) {
    this.split.notify('end', gutterNum);
  }
```

**Comment by beeman** [comment]
I've looked into this issue and confirm the behavior, the 'end' event is only emitted when the gutter changes position.

However, currently I'm not sure what's the best way to implement a fix without breaking other behavior.

Thanks to @eranshmil's example, I've now added this behavior to the iframe example page, it should land in https://github.com/bertrandg/angular-split/pull/249.

Going to close the issue as the above workaround is clean and works.

If there is a stable way to implement this into the library I'm happy to consider a PR.
